src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';');
+
+  const finalResult = {};
```

Comment:
[CODE STYLE] - The variable `finalResult` is also too generic. A name like `styleObject` would better communicate the purpose of this variable, which is to hold the final object representation of the styles.